include/AL/alc.h

```diff
@@ -148,156 +148,191 @@ typedef void ALCvoid;
 /** Out of memory. */
 #define ALC_OUT_OF_MEMORY                        0xA005
 
-
 /** Runtime ALC major version. */
 #define ALC_MAJOR_VERSION                        0x1000
+
 /** Runtime ALC minor version. */
 #define ALC_MINOR_VERSION                        0x1001
 
 /** Context attribute list size. */
 #define ALC_ATTRIBUTES_SIZE                      0x1002
+
 /** Context attribute list properties. */
 #define ALC_ALL_ATTRIBUTES                       0x1003
 
 /** String for the default device specifier. */
 #define ALC_DEFAULT_DEVICE_SPECIFIER             0x1004
+
 /**
  * Device specifier string.
  *
  * If device handle is NULL, it is instead a null-character separated list of
  * strings of known device specifiers (list ends with an empty string).
  */
 #define ALC_DEVICE_SPECIFIER                     0x1005
+
 /** String for space-separated list of ALC extensions. */
 #define ALC_EXTENSIONS                           0x1006
 
-
-/** Capture extension */
-#define ALC_EXT_CAPTURE 1
-/**
- * Capture device specifier string.
- *
- * If device handle is NULL, it is instead a null-character separated list of
- * strings of known capture device specifiers (list ends with an empty string).
- */
-#define ALC_CAPTURE_DEVICE_SPECIFIER             0x310
-/** String for the default capture device specifier. */
-#define ALC_CAPTURE_DEFAULT_DEVICE_SPECIFIER     0x311
-/** Number of sample frames available for capture. */
-#define ALC_CAPTURE_SAMPLES                      0x312
-
-
-/** Enumerate All extension */
-#define ALC_ENUMERATE_ALL_EXT 1
-/** String for the default extended device specifier. */
-#define ALC_DEFAULT_ALL_DEVICES_SPECIFIER        0x1012
-/**
- * Device's extended specifier string.
- *
- * If device handle is NULL, it is instead a null-character separated list of
- * strings of known extended device specifiers (list ends with an empty string).
- */
-#define ALC_ALL_DEVICES_SPECIFIER                0x1013
-
-
-#ifndef ALC_NO_PROTOTYPES
+#ifndef AL_NO_PROTOTYPES
```

Comment:
Not sure what's causing this, but this is a breaking change, hiding the standard ALC functions with `AL_NO_PROTOTYPES` instead of `ALC_NO_PROTOTYPES`.